treeherder/autoclassify/detect_intermittents.py

```diff
@@ -1,8 +1,8 @@
 import logging
```

Comment:
Given that we aren't running this, I feel like it might be better just to remove it rather than continuing to maintain code that we're not really testing.